### PR TITLE
chore(web): add popover contents to left panel and related modification

### PR DESCRIPTION
**Pull request by isoppp**
# Overview

~* This PR should review after https://github.com/reearth/reearth/pull/580~ it's merged now.

- Add popover menus to the page area of the left panel
- Modify the story area for 1st implementation that only has 1 story
  - This means it does not need menus and addition buttons

## How I tested

- open beta project in Editor and move to the story tab

image

- storybook
  - https://deploy-preview-581--reearth-web.netlify.app/storybook/?path=/story/beta-features-editor-tabs-story-sidepanel-contentstory--default
  - https://deploy-preview-581--reearth-web.netlify.app/storybook/?path=/story/beta-features-editor-tabs-story-sidepanel-contentpage--default

### CODE REVIEW

File: web/src/beta/features/Editor/tabs/story/SidePanel/index.tsx
Comment (KaWaite) [inline web/src/beta/features/Editor/tabs/story/SidePanel/index.tsx:1]: Can you comment out all props we won't be using until we support multiple stories? 🙏🏻

Comment (KaWaite) [inline web/src/beta/features/Editor/tabs/story/SidePanel/index.tsx:1]: Can you hide the whole Story section for now? 🙏🏻 We don't need that UI until we support multiple, so if you can revert the changes that show the buttons, etc for that section, and then just comment out its use here that would be awesome!

**Review by nina992 — APPROVED** [review]

File: web/src/beta/features/Editor/tabs/story/SidePanel/index.tsx
Comment (isoppp) [reply web/src/beta/features/Editor/tabs/story/SidePanel/index.tsx:1]: Hide and support multiple stories for future use.

- [support multiple stories](https://github.com/reearth/reearth/pull/581/commits/5a29c04bbf3a48a06c1f4a5023f500baa03116ee)
- [hide story section and update storybook stories](https://github.com/reearth/reearth/pull/581/commits/093679993f6aed0c8754e1763dce882e43e448c4)

Updated screenshot in PR description and added stories' URLs as well.

Comment (isoppp) [reply web/src/beta/features/Editor/tabs/story/SidePanel/index.tsx:1]: Thank you, fixed this. Details are the same as https://github.com/reearth/reearth/pull/581#discussion_r1270418641

**Review by KaWaite — APPROVED** [review]
